src/bindings/c/include/openvino/c/ov_rank.h

```diff
@@ -22,28 +24,30 @@ typedef struct ov_rank ov_rank_t;
  */
 
 /**
- * @brief Create a static rank object
+ * @brief Initialize a static rank object
  * @ingroup rank
- * @param rank_value The rank value for this object, it should be not less than 0(>=0)
- * @param ov_status_e a status code.
+ * @param rank The input rank pointer.
+ * @param rank_value The rank value for this object, it should be not less than 0(>=0).
+ * @return ov_status_e The return status code.
  */
-OPENVINO_C_API(ov_status_e) ov_rank_create(ov_rank_t** rank, int64_t rank_value);
+OPENVINO_C_API(ov_status_e) ov_rank_init(ov_rank_t* rank, int64_t rank_value);
 
 /**
- * @brief Create a dynamic rank object
+ * @brief Initialize a dynamic rank object.
  * @ingroup rank
- * @param min_rank The lower inclusive limit for the rank
- * @param max_rank The upper inclusive limit for the rank
- * with min_dimension
- * @param ov_status_e a status code.
+ * @param rank The input rank pointer.
+ * @param min_rank The lower inclusive limit for the rank.
+ * @param max_rank The upper inclusive limit for the rank.
+ * @return ov_status_e The return status code.
  */
-OPENVINO_C_API(ov_status_e) ov_rank_create_dynamic(ov_rank_t** rank, int64_t min_rank, int64_t max_rank);
+OPENVINO_C_API(ov_status_e) ov_rank_init_dynamic(ov_rank_t* rank, int64_t min_rank, int64_t max_rank);
```

Comment:
why user just cannot write:
```c++
ov_rank_t rank = { 1, 2 };
```
w/o any helper methods?

Reply:
I'm hesitating whether keep it, it is a simple C style structure not need helper function at all, but directly fill it may bring confusion to user on ov_rank_t itself.

Reply:
Done - Remove such unnecessary methods.